webapp/tests/karma/ts/services/db.service.spec.ts

```diff
@@ -592,19 +597,18 @@ describe('Db Service', () => {
       }));
 
       it('should catch errors', fakeAsync(async () => {
-        const target = window.PouchDB(`db-${uuidv4()}`);
         const opts = { live: false, retry: false };
 
         getService();
         const db = service.get();
         const onError = sinon.stub();
         const onComplete = sinon.stub();
-        await target.bulkDocs([{ _id: uuidv4() }]);
-        sinon.stub(target, 'allDocs').rejects({ status: 400, name: 'forbidden' });
+        await syncTarget.bulkDocs([{ _id: uuidv4() }]);
+        sinon.stub(syncTarget, 'bulkGet').rejects({ status: 400, name: 'forbidden' });
```

Comment:
Hm, did the internal sync implementation change?

Reply:
I tried to check the sync implementation (node-modules) because it wasn't breaking with rejection from `allDocs`, but `bulkGet ` breaks it. I need to research more